<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorComposer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="20,0">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="fileChooseLabel">
        <property name="text">
         <string>File List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="fileWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="addFileButton">
        <property name="text">
         <string>Add File...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="doTransformationButton">
        <property name="text">
         <string>Transformation</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cursor Radius:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="cursorRadiusSlider">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cursorRadiusSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="statsNumber">
        <property name="text">
         <string>Statistics: no beads</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statsIntensity">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOutButton">
        <property name="text">
         <string>zoom -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomInButton">
        <property name="text">
         <string>zoom +</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_Composed_image"/>
    <addaction name="actionExport_as_stack_of_images"/>
    <addaction name="actionExport_transformed_coordinates"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTransformation_Matrix"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionAuto_detect_beads"/>
    <addaction name="actionClear_all"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuExtras"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTransformation_Matrix">
   <property name="text">
    <string>Transformation Matrix...</string>
   </property>
  </action>
  <action name="actionExport_Composed_image">
   <property name="text">
    <string>Export Composed image...</string>
   </property>
  </action>
  <action name="actionExport_as_stack_of_images">
   <property name="text">
    <string>Export as stack of images...</string>
   </property>
  </action>
  <action name="actionAuto_detect_beads">
   <property name="text">
    <string>Auto-detect beads</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Discard loaded images (Reset)</string>
   </property>
  </action>
  <action name="actionExport_transformed_coordinates">
   <property name="text">
    <string>Export transformed Coordinates as List...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
